src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs

```diff
@@ -10,9 +12,6 @@ namespace System.Text.Json.Serialization
 {
     internal abstract class JsonPropertyInfo
     {
-        // For now, just a global converter.
-        private static JsonEnumerableConverter s_jsonEnumerableConverter = new DefaultArrayConverter();
```

Comment:
Why did we remove this static?

Reply:
A new `JsonEnumerableConverter`-derived class has been added so we now determine which converter to assign `EnumerableConverter` to further down in the file.  There's no real reason for this static to be singled out here.